ntpd-rs.rc.d.example

```diff
@@ -1,35 +1,23 @@
-#!/bin/sh 
+#!/bin/sh
 
-. /etc/rc.subr 
+. /etc/rc.subr
 
+# PROVIDE: ntp_daemon
 # REQUIRE: DAEMON FILESYSTEMS devfs
 # BEFORE:  LOGIN
 # KEYWORD: nojail resume shutdown
 
-name="ntp_daemon" 
+name="ntp_daemon"
 rcvar="ntp_daemon_enable"
 
+: ${ntp_daemon_enable:="NO"}
 
-# start_cmd="${name}_start" 
-stop_cmd=":" 
 start_precmd="${name}_precmd"
 
-command="/usr/local/bin/ntp-daemon"
-pidfile="/var/run/ntpd-rs.pid"
-required_files="$command"
+command="/usr/sbin/daemon"
+procname="daemon"
```

Comment:
I would set this to ntp-daemon, since you may have multiple rc scripts on a single system that use daemon.